docs/stable/intro.md

```diff
@@ -9,6 +9,10 @@ sidebar_position: 1
 
 ServerlessLLM is a **fast** and **easy-to-use** serving system designed for **affordable** multi-LLM serving, also known as LLM-as-a-Service. ServerlessLLM is ideal for environments with multiple LLMs that need to be served on limited GPU resources, as it enables efficient dynamic loading of LLMs onto GPUs. By elastically scaling model instances and multiplexing GPUs, ServerlessLLM can significantly reduce costs compared to traditional GPU-dedicated serving systems while still providing low-latency (Time-to-First-Token, TTFT) LLM completions.
 
+ServerlessLLM now supports NVIDIA and AMD GPUs, we have tested ServerlessLLM on the following hardware:
+* NVIDIA GPUs: Compute Capability 7.0+ (e.g, V100, A100, RTX A5000, RTX A6000)
```

Comment:
I think it's fine to mention just one GPU per generation. For example, listing the A100 should cover the A5000/6000. It might also be worth noting that we support RTX 30XX. As for the V100, I haven’t tested it myself, but if Siyang has, it should be fine to include it here.

Reply:
V100 is fine. Some of our dev work and test work on sllm with dual GPUs are done on V100 machine.